products/fortios.md

```diff
@@ -0,0 +1,36 @@
+---
+title: FortiOS
+layout: post
+category: os
+sortReleasesBy: "releaseCycle"
+iconSlug: fortinet
+permalink: /fortios
+activeSupportColumn: true
+releaseColumn: false
+releaseDateColumn: true
+eolColumn: End of Engineering Support Date (EOES)
+discontinuedColumn: false
+versionCommand: get system status
+releases:
+  - releaseCycle: "7.2"
+    eol: 2026-09-30
+    support: 2025-03-31
+    releaseDate: 2022-03-31
+  - releaseCycle: "7.0"
+    eol: 2026-09-30
+    support: 2025-09-30
+    releaseDate: 2021-03-30
+  - releaseCycle: "6.4"
+    eol: 2024-09-30
+    support: 2023-03-31
+    releaseDate: 2020-03-31
+  - releaseCycle: "6.2"
+    eol: 2023-09-28
+    support: 2022-03-28
+    releaseDate: 2019-03-28
+  - releaseCycle: "6.0"
+    eol: 2022-09-29
+    support: 2021-03-29
+    releaseDate: 2018-03-29
+
+---
```

Comment:
Please add description and other important details. Things to cover are cadence, versioning, upgrading, support policy summary or any other important links. 

See the Contributing doc for tone, and format. There's also another fortinet page that might be helpful.